api/server/runner.go

```diff
@@ -44,8 +61,6 @@ func (s *Server) handleRequest(c *gin.Context, enqueue models.Enqueue) {
 	ctx := c.MustGet("ctx").(context.Context)
 
 	reqID := uuid.NewV5(uuid.Nil, fmt.Sprintf("%s%s%d", c.Request.RemoteAddr, c.Request.URL.Path, time.Now().Unix())).String()
-	c.Set("reqID", reqID) // todo: put this in the ctx instead of gin's
```

Comment:
The async job receives it through the task object.
No other place in the code do a c.Get("reqID")